drivers/scheduler/k8s/specs/postgres/px-postgres-app.yaml

```diff
@@ -8,7 +8,7 @@ spec:
       maxSurge: 1
       maxUnavailable: 1
     type: RollingUpdate
-  replicas: 2
```

Comment:
here postgress is deployment having replicas 2 but volume is not shared. this is giving error for few of the tests where postgress app is used.
If this is intentional, then the test is unique and this should be parameterized in the test defaulting value to 1 only so that remaining tests are not effected